Setup/shield-version.txt

```diff
@@ -1,5 +1,5 @@
-#Build Staging:Build_485 on 19/02/13   #UNR#
-SHIELD_VER=8.0.0.latest SHIELD_VER=Staging:Build_485
+#Build Staging:Build_492 on 19/02/27   #UNR#
```

Comment:
Consider removing #UNR#...
The update will be more simple

Reply:
I agree, I dont see a reason to restart